source/operations/install-deploy-manage/deploy-operator-helm.rst

```diff
@@ -0,0 +1,410 @@
+.. _minio-k8s-deploy-operator-helm:
+
+=========================
+Deploy Operator With Helm
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+Helm is a tool for automating the deployment of applications to Kubernetes clusters.
+A `Helm chart <https://helm.sh/docs/topics/charts/>`__ is a set of YAML files, templates, and other files that define the deployment details.
+The following procedure uses a Helm Chart to install the :ref:`MinIO Kubernetes Operator <minio-operator-installation>` to a Kubernetes cluster.
+
+
+Prerequisites
+-------------
+
+To install the Operator with Helm you will need the following:
+
+* An existing cluster using Kubernetes 1.19.0 or later (1.21.0 or later recommended).
```

Comment:
https://endoflife.date/kubernetes

Kubernetes has a *really* aggressive EOL schedule. Even so, I would suggest we maybe make this a replacement variable and use it consistently in our docs (`|recommended-k8s-version|` or something) and always pin to the one with active support (1.25 right now)

Reply:
I guess we could parse the json version of that page. And by _we_ I mean someone who can operate `jq` better than I.

Reply:
Reworked text to send readers to the Prerequisites link for k8s version. Programmatic substitution can come later.